### ISSUE TITLE
Not downloading in epub format

### ISSUE THREAD

**Issue by Traveller007** [MakeYourLifeEasier/novel-ebook.com#99]
So my problem is the novels r not getting downloaded in epub format anymore the problem I read the novels using @voice app i can't do that anymore so I'd like to request to be able to download in epub or pdf format before it downloaded in epub format now it downloads in xhtml so yeah is this a temporary thing or permanent
![Screenshot_2020-12-18-13-51-32-064_com mi android globalFileexplorer](https://user-images.githubusercontent.com/76201892/102591744-87f6bb00-4138-11eb-894a-50dc820d1f66.jpg)
![Screenshot_2020-12-18-13-52-00-813_com mi android globalFileexplorer](https://user-images.githubusercontent.com/76201892/102591765-9218b980-4138-11eb-8b25-a5f311db9964.jpg)
![Screenshot_2020-12-18-13-52-04-531_com miui home](https://user-images.githubusercontent.com/76201892/102591784-99d85e00-4138-11eb-917e-5f80871bb4da.jpg)

**Comment by MichaelGuardian** [comment]
If you look closely, these are still downloaded as epub files. Epub files are essentially just zip files and the chapter content is saved in xhtml. So in the first screenshot you just went inside the epub file, like you would with a zip file. Most likely you accidentally changed the default app for .epub files. [This is how you change that](https://www.androidauthority.com/change-default-apps-android-2-633572/), if you still have problems, try reinstalling the ereader.

**Comment by Traveller007** [comment]
The problem is this↓
![Screenshot_2020-12-18-14-18-58-999_com hyperionics avar](https://user-images.githubusercontent.com/76201892/102594515-508a0d80-413c-11eb-8801-20f6add692e9.jpg)
When I open using the app it comes like this and before it was fine like this↓
![Screenshot_2020-12-18-14-20-24-879_com hyperionics avar](https://user-images.githubusercontent.com/76201892/102594611-6c8daf00-413c-11eb-9a4b-eb92c43a17fb.jpg)
And It is also not merged so I have to open every file 1 by 1 If i want to read them that's a pain too

**Comment by MichaelGuardian** [comment]
Hm, I just downloaded a copy of Overgeared and checked the file. It is still an .epub file and it's a valid version, so in theory it should be displayed properly. What you're viewing is the xhtml file not the epub file, so it might not have been added to your e reader properly or like I said before, your file explorer thinks of the .epub file as a .zip file and opened it instead of importing it to your e reader.

No matter what this seems to be either a problem on your and or a bug in the e reader of your choosing, as the file seems to be in order from my testing.
